tests/markdown_table/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -61,11 +61,31 @@ exports[`html.md 1`] = `
+| :---: | :---: | :---: | :---: |
+| a | b | c | d |
+
+| test | test | test |
+| - | - | - |
+| something something something something | something something | something something something |
+| <a href="#">something something something something</a> | something something | something something something |
+| something something something | something something something | something something |
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-| Default  | CLI Override                                           | API Override                                           |
-| -------- | ------------------------------------------------------ | ------------------------------------------------------ |
+| Default | CLI Override | API Override |
+| ------- | ------------ | ------------ |
 | \`"none"\` | <code>--trailing-comma <none&#124;es5&#124;all></code> | <code>trailingComma: "<none&#124;es5&#124;all>"</code> |
 
+| [<img src="https://avatars1.githubusercontent.com/u/8344688?v=4" width="100px;"/><br /><sub><b>Roy Revelt</b></sub>](https://github.com/revelt)<br /> [💻](https://github.com/codsen/detergent/commits?author=revelt "Code") [📖](https://github.com/codsen/detergent/commits?author=revelt "Documentation") [⚠️](https://github.com/codsen/detergent/commits?author=revelt "Tests") | [<img src="https://avatars3.githubusercontent.com/u/1874682?v=4" width="100px;"/><br /><sub><b>Max Lapides</b></sub>](https://github.com/maxlapides)<br /> [🐛](https://github.com/codsen/detergent/issues?q=author%3Amaxlapides "Bug reports") | [<img src="https://avatars3.githubusercontent.com/u/9990521?v=4" width="100px;"/><br /><sub><b>Nacim Goura</b></sub>](https://github.com/nacimgoura)<br /> [🐛](https://github.com/codsen/detergent/issues?q=author%3Anacimgoura "Bug reports") | [<img src="https://avatars2.githubusercontent.com/u/23390212?v=4" width="100px;"/><br /><sub><b>Jack Steam</b></sub>](https://github.com/jacksteamdev)<br /> [🐛](https://github.com/codsen/detergent/issues?q=author%3Ajacksteamdev "Bug reports") |
+| :-: | :-: | :-: | :-: |
+|  a  |  b  |  c  |  d  |
+
+| test                                    | test                          | test                          |
+| --------------------------------------- | ----------------------------- | ----------------------------- |
+| something something something something | something something           | something something something |
+| <a href="#">something something something something</a> | something something | something something something |
```

Comment:
In this case, the HTML is not particularly long, so the alignment looks odd. I think we might need a more fine-grained heuristic than just "contains HTML".

Reply:
It's somewhat odd, but better than completely minimal column length, isn't it? I think one of the main points is comparing the length of the current row that contains HTML (edge-to-edge) in one cell on that row AND the length of the whole result, if we gave a green light and extended it.

ESLint's [max-len](https://eslint.org/docs/rules/max-len) default value is `80`. Therefore, it's a good guide - if any of the cleaned lines exceed `80` because of cleaning, that cleaning should be undo'ed.

What do you think?

Reply:
That sounds reasonable, but I think we should use printWidth instead of 80.